### PR TITLE
RATEST-95:Fix Edit Form Test

### PR DESCRIPTION
**Pull request by HerbertYiga**
Ticket Id:https://issues.openmrs.org/browse/RATEST-95

Description 
Fix Edit Form Test

### CODE REVIEW

**Comment by sherrif10** [comment]
Same as here, it would automatically pass because it is considered as ignored, so try un ingoring it and push again thanks

File: ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java [hunk 1]
```diff
@@ -32,9 +32,8 @@ public void setUp() throws Exception {
         patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
     }
 
-    @Ignore//ignore due to moving forms functionality
     @Test
-    public void EditFormTest() throws Exception {
+    public void EditFormTest() {
```
Comment (ellieMayVelasquez) [inline ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java:36]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Comment (HerbertYiga) [reply ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java:36]: i will have to close this and make a better pr for this ticket

Comment (ellieMayVelasquez) [reply ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java:36]: Noticed :)

**Pull request closed without merging**